lib/pages/profile_page/get_profile.dart

```diff
@@ -275,43 +305,113 @@ class _GetProfileWidgetState extends State<GetProfileWidget> {
+                                                                      communityName:
+                                                                          _viewUser.communities![index][0]
+                                                                              [
+                                                                              'communityName'],
+                                                                      token: widget
+                                                                          .token,
                                                                     ),
                                                                   ),
                                                                 );
                                                               },
                                                               child: Text(
                                                                 "c/${_viewUser.communities![index][0]['communityName']}",
-                                                                style: const TextStyle(
-                                                                  fontSize: 16.0,
-                                                                  fontWeight: FontWeight.w800,
+                                                                style:
+                                                                    const TextStyle(
+                                                                  fontSize:
+                                                                      16.0,
+                                                                  fontWeight:
+                                                                      FontWeight
+                                                                          .w800,
                                                                 ),
                                                               )),
-                                                          if (_viewUser.username == _viewUser.communities![index][1]['creator'])
-                                                            Icon(
-                                                              Icons.admin_panel_settings,
-                                                              color: AppThemes.iconColor(context),
-                                                              size: 30.0,
-                                                            ),
+                                                          Row(
+                                                            mainAxisAlignment:
+                                                                MainAxisAlignment
+                                                                    .end,
+                                                            children: [
+                                                              if (computeStreak(
```

Comment:
same comment about calling `computeStreak()` once and `first_streak_date` and `last_streak_date` potentially being null

Reply:
2. First and last streak will never be null.